lib/diagnostic.rb

```diff
@@ -12,28 +12,39 @@
 # Name should be readable and writeable, age should only be readable,
 # and location should only be writable.
 
-##
-# your code here
-##
+class Person
+  attr_reader :age
+  attr_accessor :name
+  attr_writer :location
+
+  def initialize(name, age, location)
+    @name = name
+    @age = age
+    @location = location
+  end
+end
 
 # Question 2
 # Instatiate a new Person object using arguments "Dave", 32, and "Ohio".
 # Then, in the next line, change this Person's location property to "Somerville".
 # Finally, assign the modified Person to `Answer.dave` below.
 
 ##
-# your answers here
-Answer.dave = nil
+dave = Person.new('Dave', 32, 'Ohio')
+dave.location = 'Somerville'
+Answer.dave = ['Dave', 32, 'Somerville']
```

Comment:
Correct, or since you've assigned the variable `dave` to be that, you could say `Answer.dave = dave`